hugo/content/prometheus/_index.md

```diff
@@ -89,19 +79,33 @@ The following pgmonitor configuration files should be placed according to the fo
 The Alertmanager data directory should be `/var/lib/ccp_monitoring/alertmanager` and owned by the `ccp_monitoring` user.  You can set it up with:
 
 ```bash
-sudo mkdir /var/lib/ccp_monitoring/alertmanager
-sudo chmod 0700 /var/lib/ccp_monitoring/alertmanager
-sudo chown ccp_monitoring /var/lib/ccp_monitoring/alertmanager
+sudo install -d -m 0700 -o ccp_monitoring -g ccp_monitoring /var/lib/ccp_monitoring/alertmanager
 ```
 
-The following pgmonitor configuration files should be placed according to the following mapping:
+The following pgMonitor configuration files should be placed according to the following mapping:
 
-| pgmonitor Configuration File | System Location |
+| pgMonitor Configuration File | System Location |
 |------------------------------|-----------------|
 | crunchy-alertmanager-service-el7.conf | `/etc/systemd/system/alertmanager.service.d/crunchy-alertmanager-service-el7.conf`  |
 | sysconfig.alertmanager | `/etc/sysconfig/alertmanager` |
 
 
+#### Windows Server 2012R2
+
+There are Windows Servfer 2012R2 packages available to [Crunchy Data](https://www.crunchydata.com) customers who contact Crunchy Data directly.
+
+If you install the below available packages, you can continue reading at the [Setup](#setup-windows-server-2012r2) section.
```

Comment:
"If you install the below available packages" -> "If you install the below package" ?

Reply:
After installing via these RPMs, you can